### ISSUE TITLE
Promtool test rules - out of order sample on v2.6.0 but worked on v.2.5.0

### ISSUE THREAD

**Issue by kentsanggds** [prometheus/prometheus#5064]
## Bug Report
**What did you do?**
Ran `promtool test rules <rule test file>`

Rules file - https://github.com/alphagov/prometheus-aws-configuration-beta/blob/4dfb2058b805bb6dee32eb4fbc129d4be14bf70c/terraform/modules/app-ecs-services/config/alerts-tests/test-observe-alerts.yml

**What did you expect to see?**

Tests passing, they were passing on Prometheus v2.5.0

**What did you see instead? Under which circumstances?**

Tests failed with - `out of order sample`

**Environment**

* System information:

	insert output of `uname -srm` here

`Darwin 17.7.0 x86_64`

* Prometheus version:

	insert output of `prometheus --version` here

```
prometheus, version 2.6.0 (branch: non-git, revision: non-git)
  build user:       brew@HighSierra.local
  build date:       20181220-02:53:59
  go version:       go1.11.4
```

**Comment by codesome** [comment]
https://github.com/prometheus/prometheus/pull/4851 might possible be the reason, not sure, will have a look at it soon.

**Comment by kentsanggds** [comment]
has this been fixed?

**Comment by codesome** [comment]
@kentsanggds Yes, [the fix](https://github.com/prometheus/prometheus/pull/5069) is included in [`v2.6.1`](https://github.com/prometheus/prometheus/releases/tag/v2.6.1) (released yesterday).

**Comment by kentsanggds** [comment]
Great, will ty it out
